Kernel/Graphics/VirtIOGPU/GraphicsAdapter.cpp

```diff
@@ -129,6 +129,11 @@ VirtIOGraphicsAdapter::VirtIOGraphicsAdapter(PCI::DeviceIdentifier const& device
     : VirtIO::Device(device_identifier)
     , m_scratch_space(move(scratch_space_region))
 {
+    m_active_context_ids.with([](Bitmap& active_context_ids) {
```

Comment:
Please don't allocate inside a constructor, but rather do the allocation in the static factory method and then use `move` semantics to move ownership over the allocated buffer.